examples/shapes/arc.rb

```diff
@@ -0,0 +1,8 @@
+Shoes.app(height:600) do
+  arc 400, 0, 120, 100, 175, 175
+  arc 50, 50, 120, 100, 60, 225
```

Comment:
Nice! I like having multiples in example tests

Can we also test it renders inside something like a stack?

And I haven't looked at the main manual but does arc support color or stroke as well?

Reply:
(I would've gone through the manual link but on mobile at the second)

Reply:
it does support color and stroke but more like

```
shoes.app do

fill "color"
stroke "color"
arc...

end
```

Reply:
so its not something inside arc i assume

Reply:
oh yeah i will check abt tht stack thing

Reply:
yeah it does , should i update example with stack?